gateway/webhook/webhook.go

```diff
@@ -179,25 +174,36 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 		r.Header.Set("Content-Type", "application/json")
 	}
 
-	done := make(chan webhookErrorRespT)
+	done := make(chan transformerResponseT)
```

Comment:
This would be a good chance to rename `transformerResponseT` to `transformerResponse` now that we are refactoring it. We have agreed that the `T` suffix is not something we should follow when naming structs

Reply:
Done